apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/controller/PrefixPathController.java

```diff
@@ -0,0 +1,18 @@
+package com.ctrip.framework.apollo.portal.controller;
+
+import org.springframework.beans.factory.annotation.Value;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class PrefixPathController {
+
+  @Value("${prefixPath: }") private String prefixPath;
```

Comment:
`@Value("${prefixPath: }")` will cause the default prefixPath to be `" "` instead of `""`.
Though it is still working(I guess the browser does some trick), I would suggest we change it as follows:

```java
@Value("${prefixPath:}")
private String prefixPath;
```

Reply:
Basically, it works as expected. And a [FAQ](https://github.com/ctripcorp/apollo/wiki/%E9%83%A8%E7%BD%B2&%E5%BC%80%E5%8F%91%E9%81%87%E5%88%B0%E7%9A%84%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98#16-portal%E6%8C%82%E8%BD%BD%E5%88%B0nginxslb%E5%90%8E%E5%A6%82%E4%BD%95%E8%AE%BE%E7%BD%AE%E7%9B%B8%E5%AF%B9%E8%B7%AF%E5%BE%84) is added for this function.

Reply:
已按照建议进行修改